fdbclient/ManagementAPI.actor.h

```diff
@@ -61,7 +61,8 @@ class ConfigurationResult {
 		NOT_ENOUGH_WORKERS,
 		REGION_REPLICATION_MISMATCH,
 		DCID_MISSING,
-		SUCCESS
+		SUCCESS,
+		LOCKED_NOT_NEW,
```

Comment:
I think `SUCCESS` should stay the last entry in this enum. Can you please switch those two?

Reply:
Done